Directory.Packages.props

```
@@ -10,7 +10,7 @@
     
     <!-- Packages only used in the solution, upgrade at will -->
     <PackageVersion Include="BenchmarkDotNet" Version="0.13.1" />
-    <PackageVersion Include="GitHubActionsTestLogger" Version="1.3.0" />
+    <PackageVersion Include="GitHubActionsTestLogger" Version="1.4.1" />
```

Hey @NickCraver, can you test out the new alpha I made?
Sorry it took so long, dotnet test platform is hellish to work with in all honesty (and the war doesn't help either).

```suggestion
    <PackageVersion Include="GitHubActionsTestLogger" Version="2.0.0-alpha" />
```

---

Absolutely - thanks for this! Giving it a spin now